scapy/as_resolvers.py

```diff
@@ -88,7 +88,7 @@ def resolve(self, *ips):
         for l in r.splitlines()[1:]:
             if "|" not in l:
                 continue
-            asn,ip,desc = map(str.strip, l.split("|"))
+            asn, ip, desc = [e for e in l.strip().split('|')]
```

Comment:
Warning: this is quite different from the original code. You may want to write: `[elt.strip() for elt in l.split('|')]`.